prober/icmp.go

```diff
@@ -119,10 +139,23 @@ func ProbeICMP(ctx context.Context, target string, module config.Module, registr
 		if srcIP == nil {
 			srcIP = net.ParseIP("0.0.0.0")
 		}
-		icmpConn, err := net.ListenPacket("ip4:icmp", srcIP.String())
-		if err != nil {
-			level.Error(logger).Log("msg", "Error listening to socket", "err", err)
-			return
+
+		var icmpConn *icmp.PacketConn
+		if tryUnprivileged && !module.ICMP.DontFragment {
```

Comment:
Why the DontFragment?

Reply:
Because IP options can't be set by unprivileged sockets, comment added.

Reply:
This isn't mentioned in the readme.